src/sample_data/Samples.js

```diff
@@ -0,0 +1,36 @@
+const Samples = {
```

Comment:
Because this is data, and not a React.Component, the convention is `camelCase`, as opposed to `CaptialCamelCase`.  Because of an idiosyncrasy with how git works and filenaming, I suggest renaming this file to `sampleSummaryStatistics.js`, instead of `samples.js`.